src/test/java/uk/gov/hmcts/reform/ccd/documentam/controller/endpoints/CaseDocumentAmControllerTest.java

```diff
@@ -632,11 +633,11 @@ public void shouldThrowBadRequestExceptionWhenJurisdictionIdIsMalformed() {
         ));
     }
 
-    private ResponseEntity<StoredDocumentHalResource> setDocumentMetaData() {
+    private Optional<StoredDocumentHalResource> setDocumentMetaData() {
```

Comment:
If this method just returns a `StoredDocumentHalResource`, then we don't need the confusing calls of `setDocumentData().get()` throughout this class?

If there is somewhere we need an optional we can use `Optional.of` ?